ui/src/app/components/html-viewer.tsx

```diff
@@ -4,9 +4,13 @@ import { SpinnerOverlay } from 'app/components/spinners';
 import { withWindowSize } from 'app/utils';
 import { WindowSizeProps } from 'app/utils';
 
+// for query selection of the last document element
+const MS_WORD_PARAGRAPH_CLASS = '.MsoNormal';
```

Comment:
Do we have a test that covers this? i.e. if someone in the future tries to create the DUCC via some other mechanism, and the HTML no longer contains this class - will there be a unit test failure?

Reply:
we do not, and I have not yet come up with any decent ideas for how to unit-test this.

e2e has a TOS test which seems likely to cover this scenario, so maybe we could reuse that for DUCC.

Reply:
Adding a DUCC e2e test to an epic we're working on soon: https://precisionmedicineinitiative.atlassian.net/browse/RW-7911

Reply:
I'd suggest just a simple unit test that asserts that the DUCC contains an element with this class - just something that breaks obviously.